archunit/src/test/java/com/tngtech/archunit/library/freeze/ViolationLineMatcherFactoryTest.java

```diff
@@ -39,11 +39,11 @@ public class ViolationLineMatcherFactoryTest {
                     "A:1|" +
                     "A$2|" + false,
             "" +
-                    "Method <MyClass.lambda$myFunction$2()> has a violation in (MyClass.java:123)|" +
-                    "Method <MyClass.lambda$myFunction$123()> has a violation in (MyClass.java:0)|" + true,
+                    "Method <MyClass.access$myFunction$2()> has a violation in (MyClass.java:123)|" +
```

Comment:
`access$` is principally still realistic, because the compiler creates these methods to bridge visibility for inner classes (e.g. if an outer class directly accesses the private field of an inner class). However, AFAIK these methods follow the naming schema `access$000` i.e. one dollar followed by three digits :thinking: So I'll adjust it to match that pattern to be more realistic...